Traceback (most recent call last):
  File "test_paralell.py", line 278, in <module>
    play(args) #, nn_path=os.path.join(settings.models_dir,"rl39699.pth"))
  File "test_paralell.py", line 81, in play
    nn.to(device)
  File "/seu_share/home/weiweiwu/anaconda3/lib/python3.8/site-packages/torch/nn/modules/module.py", line 673, in to
    return self._apply(convert)
  File "/seu_share/home/weiweiwu/anaconda3/lib/python3.8/site-packages/torch/nn/modules/module.py", line 387, in _apply
    module._apply(fn)
  File "/seu_share/home/weiweiwu/anaconda3/lib/python3.8/site-packages/torch/nn/modules/module.py", line 387, in _apply
    module._apply(fn)
  File "/seu_share/home/weiweiwu/anaconda3/lib/python3.8/site-packages/torch/nn/modules/module.py", line 387, in _apply
    module._apply(fn)
  File "/seu_share/home/weiweiwu/anaconda3/lib/python3.8/site-packages/torch/nn/modules/module.py", line 409, in _apply
    param_applied = fn(param)
  File "/seu_share/home/weiweiwu/anaconda3/lib/python3.8/site-packages/torch/nn/modules/module.py", line 671, in convert
    return t.to(device, dtype if t.is_floating_point() or t.is_complex() else None, non_blocking)
RuntimeError: CUDA error: out of memory
